os/drivers/kernel_test/kernel_test_codes/kernel_test_proto.h

```diff
@@ -34,4 +34,7 @@ int test_tz(void);
 #ifdef CONFIG_EXAMPLES_STACK_PROTECTION
 int test_kthread_stack_overflow_protection(int cmd, unsigned long arg);
 #endif
+#ifdef CONFIG_TC_NET_PBUF
```

Comment:
.c is in network folder but the prototype is in kernel folder. That's wrong.

Reply:
corrected